regulations/check_missing.py

```diff
@@ -0,0 +1,34 @@
+import os
+import subprocess
+
+import re
+MAX_LANG_WIDTH = 20
+FNULL = open(os.devnull, 'w')
+WARNING_FORMAT = ('%s - %s does not match a regulation, '
+                  'please change "%s" to "[SEPARATE]%s".\n')
+
+def check_guidelines(language, base_path):
+  padded_language = ("[" + language + "]").ljust(MAX_LANG_WIDTH +2)
+  with open(base_path + "/wca-guidelines.md") as guidelines:
+    warnings = ""
+    for item in guidelines:
+      # Only match the lines starting with something like "- A1b++)" (guidelines),
+      # which don't have the "SEPARATE" label.
+      match_result = re.match(r"- (\w+)(\++)\) (?!\[SEPARATE\])(\[[^\[\]]+\])", item)
+      if match_result is None:
+        continue
+      # Extract the regulation we want to look for
+      # '0' is the full match, '1' is the first subgroup (regulation), '2' is the subnumber
```

Comment:
I have no idea about tuple unpacking vs naming tbh, but it would indeed make the regex more readable !